external/ossl_lib/include/__common.h

```diff
@@ -0,0 +1,66 @@
+#define NULL 0
```

Comment:
Filename is ridiculous, I assume these includes are from OpTEE/UEFI/somewhere?

Reply:
This came as a package. It was based on work done for the Linux shielded VMs (https://github.com/Microsoft/lsvmtools) and was provided by the original author. I don't know the rational for the choice so I'll try swapping the name to something like "sassl_common.h" and see if it causes any issues.

Reply:
Rename seems to have worked fine.